langgraph/pregel/__init__.py

```diff
@@ -487,7 +492,9 @@ def update_state(
         saved = self.checkpointer.get_tuple(config)
         checkpoint = copy_checkpoint(saved.checkpoint) if saved else empty_checkpoint()
         # find last node that updated the state, if not provided
-        if as_node is None and not saved:
+        if as_node is None and not any(
+            v for vv in checkpoint["versions_seen"].values() for v in vv.values()
```

Comment:
is this related?

Reply:
No, just an edge case I found while migrating js